test/cli.js

```diff
@@ -110,3 +114,45 @@ test('pkg-conf: cli takes precedence', function (t) {
 		t.end();
 	});
 });
+
+test('watcher works', function (t) {
+	var killed = false;
+
+	var hasChokidar = false;
+	try {
+		require('chokidar');
+		hasChokidar = true;
+	} catch (err) {}
+
+	var child = execCli(['--verbose', '--watch', 'test.js'], 'fixture/watcher', function (err, stdout) {
+		if (err && err.code === 1 && !hasChokidar) {
+			t.comment('chokidar dependency is missing, cannot test watcher');
+			t.match(stdout, 'The optional dependency chokidar failed to install and is required for --watch. Chokidar is likely not supported on your platform.');
+			t.end();
+		} else {
+			t.comment('chokidar dependency is installed, cannot test watcher behavior with missing dependency');
```

Comment:
The test passes either way so you can't obviously tell which branch passed, hence the comment output. Though that still requires you to look at raw TAP so probably not that useful.